src/theme/Navbar/Content/index.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { useThemeConfig, ErrorCauseBoundary } from '@docusaurus/theme-common';
+import { splitNavbarItems, useNavbarMobileSidebar } from '@docusaurus/theme-common/internal';
+import NavbarItem from '@theme/NavbarItem';
+import NavbarColorModeToggle from '@theme/Navbar/ColorModeToggle';
+import SearchBar from '@theme/SearchBar';
+import NavbarMobileSidebarToggle from '@theme/Navbar/MobileSidebar/Toggle';
+import NavbarLogo from '@theme/Navbar/Logo';
+import NavbarSearch from '@theme/Navbar/Search';
+import useOfficialContentsContext from '@site/src/hooks/useOfficialContentsContext';
+import styles from './styles.module.css';
+function useNavbarItems() {
```

Comment:
Is it possible to update to ES6 functions?
we have multiple functions in this file.